deps/Stride.GitVersioning/Stride.GitVersioning.csproj

```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net472</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="16.11.0" ExcludeAssets="runtime" />
-    <PackageReference Include="LibGit2Sharp" Version="0.25.3" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="17.3.1" ExcludeAssets="runtime" />
+    <PackageReference Include="LibGit2Sharp" Version="0.27.0-preview-0182 " />
```

Comment:
Does it make sense to reference a preview version instead of a stable one?

Reply:
I've decided to choose latest available version because stable version was about 3 years ago and preview versions pretty much works. But I guess updating to 0.26.2 should be also fine.